File: pinyougou-parent/pinyougou-manager-web/src/main/java/cn/porkchop/pinyougou/manager/controller/UserController.java
package cn.porkchop.pinyougou.manager.controller;

import org.springframework.security.core.context.SecurityContextHolder;
import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.RestController;

import java.util.HashMap;
import java.util.Map;

@RestController
@RequestMapping("/user")
public class UserController {
    @RequestMapping("/getUserName")
    public Map<String, String> getUserName() {
        String name = SecurityContextHolder.getContext().getAuthentication().getName();
        Map<String, String> map = new HashMap<String, String>();
        map.put("userName", name);
        return map;
    }
}

File: pinyougou-parent/pinyougou-manager-web/src/main/webapp/js/controller/userController.js
//控制层
app.controller('userController', function ($scope, $controller, userService) {
    //获取登陆的用户名
    $scope.getUserName = function () {
        userService.getUserName().success(
            function (response) {
                $scope.userName= response.userName;
            }
        );
    };
});
